engines/bastion_katello/app/assets/stylesheets/bastion_katello/bastion_katello.scss

```diff
@@ -26,4 +26,7 @@
     text-overflow: ellipsis;
   }
 
+  .breadcrumb {
+    margin-bottom: 0px;
```

Comment:
I meant whatever the margin was before these changes, i.e. stick with what bootstrap/patternfly defined already rather than overriding it.